content/articles/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/index.md

```diff
@@ -0,0 +1,110 @@
+
+A wide range of studies have focused on detecting a person's emotions. They include the use of computer games, audio surveillance, robots, others. Machine learning algorithms have been termed as more efficient in classification and prediction, which is vital in speech emotion recognition. However, there is a lot of information about the use of [CART](/engineering-education/decision-tree-in-python/) in speech emotion recognition that needs assessment which the study focuses on.
+
+The article focuses on assessing the substantial use of CART algorithms in speech emotion recognition. In achieving the purpose, the article reviews the speech emotion system, how CART works, and various reasons affiliated with its effectiveness.
+
+### The system of emotion recognition
+
+The system of speech emotion recognition is so like the pattern system. The system automatically identifies the human beings' emotions from a voice. The system is usually based on the signal of speech and emotion recognition methods. The system consists of five main steps;
+
+- Emotional speech input
+- Feature selection and extraction
+- Training
+- Classification
+- Emotion recognition
+
+#### Emotional speech input
+
+The step entails the collection of voice samples, which will be analyzed for emotion recognition. This ensures that the vital aspects of speech that could be used in extracting emotion features are inputted. The emotional address signal must also be inputted before extraction.
+
+#### Feature extraction and selection
+
+There is a wide range of speech signal parameters that emotional characteristics reflect. The features which are extractable can be categorized in various main aspects. They include pitch, energy, formant, modulation spectral features, and linear prediction coefficients. From the features selected, the selection is made to reduce the elements which characterize the database. The chosen ones are a subset of more relevant features from the extracted ones based on the relevance evaluation criterion.
+
+The various emotional recognition extracts critical emotional features from speech. Pitch is one of the essential features, which is the sound's frequency. The features also include standard deviation, energy intensity, and shimmer. Others include zero-crossing rate, short-time energy autocorrelation, and [MFCC](https://musicinformationretrieval.com/mfcc.html#) (Mel frequency cepstral coefficient). Other features include spectral roll-off, spectral flux, noise to harmonic ratio, harmonics to noise ratio, and jitter.
+
+#### Training
+
+This is another crucial step of the speech recognition system. The step entails voice training of the computers for speech recognition. The entails entail assigning a weight of features using estimators and considering small sets of features. The estimators are trained on the features of the initial stage and predictive power.
```

Comment:
The second line _the entails entail...._. Rephrase this

Reply:
Alright, will handle them